Platform/IdavilleBoardPkg/Library/Stage2BoardInitLib/Stage2BoardInitLib.c

```diff
@@ -1020,8 +1020,8 @@ PlatformUpdateAcpiGnvs (
 {
   RETURN_STATUS             Status;
   UINT32                    IioPresentBitMask;
-  EFI_CPUID_REGISTER        CpuidRegisters;
-  UINT32                    NumOfBitShift;
+  EFI_CPUID_REGISTER        CpuidRegisters = {0};
+  UINT32                    NumOfBitShift = 0;
```

Comment:
Following edk2 coding style, variable should be initialized separately (not doing it when declaring it).

Reply:
Initialized the variable as per EDK2